delta/utils/decode/tf_ctc.py

```diff
@@ -46,8 +46,27 @@ def ctc_decode_data_prepare(logits, sequence_length, blank_id=None):
       false_fn=lambda: tf.squeeze(sequence_length),
   )
 
-  return logits_return, sequence_length_return
+  return logits_return, sequence_length_return, blank_id
 
+def ctc_decode_post_process(decode_result, blank_id):
```

Comment:
Maybe `ctc_decode_blankid_to_last`, `ctc_decode_last_to_blankid` is good for these interfaces.

Reply:
add unit test for theses two interfaces.